### PR TITLE
mouseDown touch-input NaN/Null Fix

### PR DESCRIPTION
**Pull request by bbanothu**
### Summary
 
The reason for these changes is to fix an issue with signing on windows touch screen devices. On mouseDown was not done correctly which leads to Nan Values. 

**A description of the original issue:**
esignature on the web would not allow for signing via touch on any windows based device with touch screen capabbilities (i.e surface pro). Fixed a bug an a preexisting pr. 

**Steps to reproduce the original issue:**
1. Boot up the terra dev site.
2. go into signature. 
3. add console log for the lineSegements on mouse down.
4. attempt to make two seperate lines with touch inputs and then check the values of the line segments. it contains Nan's. 

**A description of the changes that have been made, why they were made, and how they are resolving the issue:**
2. Added checks for touchInputs vs mouseInputs on mouseDown. 

**Steps to validate the changes are working as expected:**
1. Boot up the terra dev site.
2. go into signature. 
3. add console log for the lineSegements on mouse down.
4. attempt to make two seperate lines with touch inputs and then check the values of the line segments. it should no longer contain nan's. 

### Testing

This was done manually on:
Ipad, Surface pro, and Mac using safari, chrome, IE, and Edge

### Additional Details

A fix on a previous Pr -> https://github.com/cerner/terra-core/pull/3391 

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Review by vikonava — APPROVED** [review]

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 Cerner Corporation
 - Bharath Banothu [@bbanothu]
+- Bharath Banothu [@bbanothu]
```
Comment (mjhenkes) [inline CONTRIBUTORS.md:3]: This looks like a duplicate entry.

File: packages/terra-signature/src/Signature.jsx [hunk 2]
```diff
@@ -115,7 +115,11 @@ class Signature extends React.Component {
 
     this.canvasRect = this.canvas.getBoundingClientRect();
 
-    this.addLine(event.pageX - this.canvasRect.left, event.pageY - this.canvasRect.top, false);
+    if (event.type === 'touchstart') {
```
Comment (mjhenkes) [inline packages/terra-signature/src/Signature.jsx:118]: are there any tests you could write for this?

File: CONTRIBUTORS.md (hunk 1 above)
Comment (bbanothu) [reply CONTRIBUTORS.md:3]: Fixed, I thought I needed to reAdd my name again for a new merge.

File: packages/terra-signature/src/Signature.jsx (hunk 2 above)
Comment (bbanothu) [reply packages/terra-signature/src/Signature.jsx:118]: https://github.com/cerner/terra-core/pull/3436/files/61110e29625b4f0be9cdb948403e8abf0c183ce6#diff-101895006107b419f432f22c93db224b2ab5f26689e324d4a7e89f1914ed3604R104

This PR just contains one check that wasn't made during the last PR for this bug. Similar to all of these changes in mouseInBounds and mouseXY. We were not able to identify or write any of tests for this last time so I do not think there are any.

**Review by dkasper-was-taken — APPROVED** [review]

**Pull request merged**